src/notation/internal/notationactioncontroller.cpp

```diff
@@ -979,6 +980,33 @@ void NotationActionController::move(MoveDirection direction, bool quickly)
             return;
         }
 
+        if (playbackController()->isPlaying()) {
+            MeasureBeat beat = playbackController()->currentBeat();
+            int targetBeatIdx = beat.beatIndex;
+            int targetMeasureIdx = beat.measureIndex;
+            int increment = (direction == MoveDirection::Right ? 1 : -1);
+
+            const engraving::RepeatList& rl = currentMasterNotation()->masterScore()->repeatList(true);
```

Comment:
Passing `true` here will probably result in unexpected behaviour for scores that do contain repeats but have repeat playback disabled. 

I think your previous implementation using `playbackController()->beatToMilliseconds(…)` was actually good; but the bug was inside `beatToMilliseconds`.
First, it calls `notationPlayback()->beatToTick(…)`, which clearly returns "raw" ticks, i.e. not aware of repeats.
Then it passes that value to `tickToMsecs`, which calls `notationPlayback()->playedTickToSec(…)`. That clearly expects "played" ticks, i.e. repeat-aware ticks.
So that's incompatible and results in wrong values.

How to proceed?
In `PlaybackController`, I'd rename `tickToMsecs` to `playedTickToMsecs`, and `tickToSecs` to `playedTickToSecs`, to avoid confusion.
Then, I'd introduce `rawTickToSecs`, which makes use of `NotationPlayback::playPositionTickByRawTick`.
Then, I'd rewrite `PlaybackController::beatToMilliseconds` using this new method; and actually, you could create a new `beatToSecs` method as well.

Along the way, there will be a few opportunities to eliminate secs <-> msecs conversions; and a few opportunities to make naming of methods more consistent (for example, unify the "raw" vs "played" ticks terminology in PlaybackController and NotationPlayback). It would be great if you make use of these opportunities :)